src/server/middleware/helmet.ts

```diff
@@ -68,13 +69,9 @@ export const helmet = koaHelmet({
       defaultSrc,
       scriptSrc: [
         "'unsafe-eval'",
+        "'unsafe-inline'",
```

Comment:
my security heart is crying but i know we have to atm

Reply:
I feel you! 😅